### PR TITLE
1.2.0.1 v3

### PR DESCRIPTION
**Pull request by kavyainfystrat**
To improve the performance we are added the philsys 1.1.5.5 config to Mosip config 1.2.0.1 v3

### CODE REVIEW

File: kernel-default.properties [hunk 1]
```diff
@@ -279,8 +279,10 @@ mosip.kernel.prereg.secret.key=${mosip.prereg.client.secret}
 ## UIN scheduler
 kernel.uin.transfer-scheduler-type=cron
 kernel.uin.transfer-scheduler-seconds=0
-kernel.uin.transfer-scheduler-minutes=48
-kernel.uin.transfer-scheduler-hours=17
+#kernel.uin.transfer-scheduler-minutes=48
+#kernel.uin.transfer-scheduler-hours=17
```
Comment (vishwa-vyom) [inline kernel-default.properties]: Please remove the commented lines in all files

File: registration-processor-default.properties [hunk 2]
```diff
@@ -735,14 +759,16 @@ mosip.regproc.message.sender.server.servlet.path=/registrationprocessor/v1/sende
...
+#mosip.regproc.printing.eventbus.kafka.max.poll.records=100
+mosip.regproc.printing.eventbus.kafka.max.poll.records=5
 mosip.regproc.printing.eventbus.kafka.poll.frequency=100
 mosip.regproc.printing.eventbus.kafka.group.id=printing-stage
 mosip.regproc.printing.message.expiry-time-limit=${mosip.regproc.common.stage.message.expiry-time-limit}
 mosip.regproc.printing.server.port=8097
 mosip.regproc.printing.server.servlet.path=/registrationprocessor/v1/print-stage
 mosip.regproc.printing.eventbus.port=5722
 
+
```
Comment (vishwa-vyom) [inline registration-processor-default.properties]: Please remove unwanted new lines

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor-default.properties (hunk 2 above)
Comment (kavyainfystrat) [reply registration-processor-default.properties]: its done

File: registration-processor-default.properties [hunk 3]
```diff
@@ -592,7 +593,7 @@ mosip.regproc.packet.classifier.server.servlet.path=/registrationprocessor/v1/pa
 
 ## Quality classifier stage
 mosip.regproc.quality.classifier.eventbus.kafka.commit.type=single
-mosip.regproc.quality.classifier.eventbus.kafka.max.poll.records=100
+mosip.regproc.quality.checker.eventbus.kafka.max.poll.records=10
```
Comment (vishwa-vyom) [inline registration-processor-default.properties]: This should be classifer not checker

File: kernel-default.properties [hunk 4]
```diff
@@ -276,11 +276,12 @@ mosip.kernel.prereg.realm-id=mosip
 mosip.kernel.prereg.client.id=mosip-prereg-client
 mosip.kernel.prereg.secret.key=${mosip.prereg.client.secret}
 
+
```
Comment (vishwa-vyom) [inline kernel-default.properties]: Avoid adding unnecessary line

File: registration-processor-default.properties [hunk 5]
```diff
@@ -326,7 +326,8 @@ mosip.regproc.verification.server.servlet.path=/registrationprocessor/v1/verific
 
 # Manual adjudication
 mosip.regproc.manual.adjudication.eventbus.kafka.commit.type=single
-mosip.regproc.manual.adjudication.eventbus.kafka.max.poll.records=100
+#mosip.regproc.manual.adjudication.eventbus.kafka.max.poll.records=100
```
Comment (vishwa-vyom) [inline registration-processor-default.properties]: Remove the comments

File: registration-processor-default.properties [hunk 6]
```diff
@@ -620,9 +621,9 @@ mosip.regproc.introducer-validator.eventbus.port=5728
 mosip.regproc.introducer-validator.server.servlet.path=/registrationprocessor/v1/introducervalidator
 
 #demo-dedupe-stage
-mosip.regproc.demo.dedupe.eventbus.kafka.commit.type=single
-mosip.regproc.demo.dedupe.eventbus.kafka.max.poll.records=100
-mosip.regproc.demo.dedupe.eventbus.kafka.poll.frequency=100
+mosip.regproc.demo.dedupe.eventbus.kafka.commit.type=auto
```
Comment (vishwa-vyom) [inline registration-processor-default.properties]: This lets make it as single

File: registration-processor-default.properties [hunk 7]
```diff
@@ -666,7 +667,7 @@ mosip.regproc.uin.generator.server.servlet.path=/registrationprocessor/v1/uin-ge
 # abis-middle-ware-stage
 mosip.regproc.abis.middleware.eventbus.kafka.commit.type=single
 mosip.regproc.abis.middleware.eventbus.kafka.max.poll.records=100
-mosip.regproc.abis.middleware.eventbus.kafka.poll.frequency=100
+mosip.regproc.abis.middleware.eventbus.kafka.poll.frequency=30000
```
Comment (vishwa-vyom) [inline registration-processor-default.properties]: 30 sec is very high, lets make it 5 seconds

File: registration-processor-default.properties [hunk 8]
```diff
@@ -844,7 +845,6 @@ mosip.regproc.packet.classifier.tagging.exceptionbiometrics.bio-value-mapping={'
 # file Identity segment and should have proper default source configured
 mosip.regproc.packet.classifier.tagging.idobject-data-availability.availability-expression-map={'INTRODUCER_AVAILABILITY':'introducerUIN || introducerRID || introducerVID'}
 
-
```
Comment (vishwa-vyom) [inline registration-processor-default.properties]: Lets not remove additional lines

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

**Review by vishwa-vyom — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**